react/features/conference/components/web/Conference.tsx

```diff
@@ -430,4 +433,46 @@ function _mapStateToProps(state: IReduxState) {
     };
 }
 
-export default reactReduxConnect(_mapStateToProps)(translate(Conference));
+export default reactReduxConnect(_mapStateToProps)(translate(props => {
+    const dispatch = useDispatch();
+
+    const [ isDragging, setIsDragging ] = useState(false);
+
+    const isModerator = useSelector(isLocalParticipantModerator);
+    const { isOpen: isChatOpen } = useSelector((state: IReduxState) => state['features/chat']);
+
+    const handleDragEnter = useCallback((e: React.DragEvent) => {
+        e.preventDefault();
+        e.stopPropagation();
+        setIsDragging(true);
+    }, []);
+
+    const handleDragLeave = useCallback((e: React.DragEvent) => {
+        e.preventDefault();
+        e.stopPropagation();
+        setIsDragging(false);
+    }, []);
+
+    const handleDragOver = useCallback((e: React.DragEvent) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        if (isModerator && isDragging) {
+            if (!isChatOpen) {
+                dispatch(openChat());
+            }
+            dispatch(setFocusedTab(ChatTabs.FILE_SHARING));
```

Comment:
All of this should not happen if the user is not a moderator or there is no support for file sharing.